### ISSUE TITLE
robots.txt not existing for a site is really an error 

### ISSUE THREAD

**Issue by valentinedwv** [gleanerio/gleaner#144]
Is it an actual error when we don't get a robots.txt,
Need to check on other items that may be really info/trace.

 {"file":"/github/workspace/internal/summoner/acquire/resources.go:204","func":"github.com/gleanerio/gleaner/internal/summoner/acquire.getRobotsForDomain","level":"error","msg":"error getting robots.txt for https://www.earthcube.org/datasets/allgood:Robots.txt unavailable at https://www.earthcube.org/datasets/allgood/robots.txt","time":"2023-01-30T20:45:53-06:00"}
{"file":"/github/workspace/internal/summoner/acquire/resources.go:66","func":"github.com/gleanerio/gleaner/internal/summoner/acquire.ResourceURLs","level":"error","msg":"Error getting robots.txt for geocodes_demo_datasets, continuing without it.","time":"2023-01-30T20:45:53-06:00"}

**Comment by nein09** [comment]
Yeah, I'd support this moving to 'info', unless the source type is `robots`.
